control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1279,10 +1279,26 @@ func (r *HostedControlPlaneReconciler) reconcilePKI(ctx context.Context, hcp *hy
 		return fmt.Errorf("failed to reconcile kas kubelet client secret: %w", err)
 	}
 
-	// KAS aggregator cert secret
+	// KAS aggregator client signer
+	kasAggregateClientSigner := manifests.AggregateClientSigner(hcp.Namespace)
+	if _, err := createOrUpdate(ctx, r, kasAggregateClientSigner, func() error {
+		return pki.ReconcileAggregateClientSigner(kasAggregateClientSigner, p.OwnerRef)
+	}); err != nil {
+		return fmt.Errorf("failed to reconcile root CA: %w", err)
+	}
+
+	// KAS aggregator client CA
+	kasAggregateClientCA := manifests.AggregateClientCAConfigMap(hcp.Namespace)
```

Comment:
this should probably follow the next block `kasAggregatorCertSecret := manifests.KASAggregatorCertSecret(hcp.Namespace)`
where the actual aggregator secret is getting created